.github/workflows/publish.yml

```diff
@@ -9,11 +9,53 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - name: Read .nvmrc
+        run: echo "::set-output name=NODE_VERSION::$(cat .nvmrc)"
+        id: nvm
       - uses: actions/setup-node@v1
         with:
-          node-version: 10.19.0
+          node-version: ${{ steps.nvm.outputs.NODE_VERSION }}
```

Comment:
I don't think it's necessary to use the same node version as the app, because we are not building in this action.

And regarding that too (but not related to this PR): I think we can remove the `npm ci` from this action, because `npm publish` doesn't depend on it, and it should make our publishing faster.

Reply:
Oh, good call 😅